<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RuKuDialog</class>
 <widget class="QDialog" name="RuKuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>重量</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="weightLine">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLineEdit" name="countLine">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>150</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>数量</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="captureButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>22</y>
        <width>149</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>拍照</string>
      </property>
     </widget>
     <widget class="QPushButton" name="finishButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>102</y>
        <width>149</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>打开柜门</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exitButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>182</y>
        <width>149</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="text">
       <string>退出</string>
      </property>
     </widget>
     <zorder>label_2</zorder>
     <zorder>weightLine</zorder>
     <zorder>countLine</zorder>
     <zorder>label</zorder>
     <zorder>captureButton</zorder>
     <zorder>finishButton</zorder>
     <zorder>exitButton</zorder>
     <zorder>groupBox_2</zorder>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>照片显示区</string>
     </property>
     <widget class="QLabel" name="previousImage">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>30</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="cameraImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>391</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
